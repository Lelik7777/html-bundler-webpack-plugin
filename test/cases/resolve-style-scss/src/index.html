<html>
  <head>
    <title>Test</title>
    <link href="./favicon.ico" rel="icon" />
    <link href="./styles.scss" rel="alternate stylesheet">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>