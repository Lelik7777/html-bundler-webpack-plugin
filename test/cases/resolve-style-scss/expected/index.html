<html>
  <head>
    <title>Test</title>
    <link href="assets/images/favicon.697ef306.ico" rel="icon" />
    <link href="styles.css" rel="alternate stylesheet">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>